src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/MakeAnApplicationHandler.java

```diff
@@ -80,6 +89,17 @@ public boolean canHandle(PreSubmitCallbackStage callbackStage,
                 currentState.toString()
             );
 
+        String applicant = userDetailsHelper.getLoggedInUserRoleLabel(userDetails).toString();
+        UserRole applicantRole = userDetailsHelper.getLoggedInUserRole(userDetails);
+
+        waFieldsPublisher.addLastModifiedApplication(asylumCase,
+                applicant,
+                applicationType,
+                applicationReason,
+                applicationEvidence.orElse(Collections.emptyList()),
+                "Pending",
```

Comment:
is this 'P' capital?

Reply:
On creation, MakeAnApplication is defaulted with this decision - "Pending"

Reply:
Probably it's a good idea to move this hardcoded word to MakeAnApplicationDecision enum